server/src/core/app.ts

```diff
@@ -1,11 +1,18 @@
 import { UserService } from './user';
+import { TransactionService } from './payment';
 
 export interface App {
-  users: UserService
+  users: UserService,
+  transactions: TransactionService
 };
 
 export interface AppConfig {
+  port: number;
   dbUri: string;
   dbName: string;
-  port: number;
+  atUsername: string;
+  atApiKey: string;
+  atPaymentsProductName: string;
+  atPaymentsProviderChannel: string;
+  atWebhooksRoot: string;
```

Comment:
Could you explain to me what `atUsername`, `atApiKey`, `atPaymentsProductName`, `atPaymentsProviderChannel` are for? What are their role within the app?